src/providers/data_providers/businessImageProviderAggregator.ts

```diff
@@ -9,21 +9,19 @@ import { BusinessInfoInput } from '../../models/data_models/types';
 import { defaultConfig, DataSourceConfiguration } from '../../models/data_models/dataSourceConfiguration';
 
 export class BusinessImageProviderAggregator {
-	// load configs here
 	protected dataSourceConfig: DataSourceConfiguration;
 
-	constructor() {
-		this.dataSourceConfig =
-			(process.env.BUSINESS_API_CONFIG && JSON.parse(process.env.BUSINESS_API_CONFIG)) || defaultConfig;
+	constructor(protected config: DataSourceConfiguration) {
+		this.dataSourceConfig = config;
 	}
```

Comment:
I believe what you have here creates an additional property called `config` on the aggregator

```suggestion
	constructor(protected dataSourceConfig: DataSourceConfiguration) {}
```

Reply:
But `dataSourceConfig` is a protected variable for the class.